### PR TITLE
Update Util.java

### PR DESCRIPTION
**Pull request by robinshine**

### CODE REVIEW

File: componentA/src/com/example/componentA/Util.java [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package com.example.componentA;
 
 /* add some comments */
-public class Util {
```
Comment (yjshen324) [inline componentA/src/com/example/componentA/Util.java]: AAA

File: componentA/src/com/example/componentA/Util.java [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+public class MyUtil {
 	public static boolean isBlank(String text) {
 		return text == null || text.trim().length() == 0;
 	}
```
Comment (yjshen324) [inline componentA/src/com/example/componentA/Util.java]: BBB

File: componentA/src/com/example/componentA/Util.java (hunk 1 above)
Comment (robinshine) [reply componentA/src/com/example/componentA/Util.java]: AAA1

File: componentA/src/com/example/componentA/Util.java (hunk 2 above)
Comment (robinshine) [reply componentA/src/com/example/componentA/Util.java]: BBB1

File: componentA/src/com/example/componentA/Util.java (hunk 1 above)
Comment (robinshine) [reply componentA/src/com/example/componentA/Util.java]: AAA2

File: componentA/src/com/example/componentA/Util.java (hunk 2 above)
Comment (yjshen324) [reply componentA/src/com/example/componentA/Util.java]: BBB2

File: componentA/src/com/example/componentA/Util.java (hunk 1 above)
Comment (robinshine) [reply componentA/src/com/example/componentA/Util.java]: AAA3

File: componentA/src/com/example/componentA/Util.java (hunk 2 above)
Comment (yjshen324) [reply componentA/src/com/example/componentA/Util.java]: BBB4

Comment (robinshine) [reply componentA/src/com/example/componentA/Util.java]: BBB5

File: componentA/src/com/example/componentA/Util.java (hunk 1 above)
Comment (robinshine) [reply componentA/src/com/example/componentA/Util.java]: AAA4

File: componentA/src/com/example/componentA/Util.java [hunk 3]
```diff
@@ -1,7 +1,7 @@
 package com.example.componentA;
```
Comment (yjshen324) [inline componentA/src/com/example/componentA/Util.java]: 111111

File: componentA/src/com/example/componentA/Util.java [hunk 4]
```diff
@@ -1,7 +1,7 @@
...
 
 /* add some comments */
-public class Util {
+public class MyUtil {
 	public static boolean isBlank(String text) {
 		return text == null || text.trim().length() == 0;
```
Comment (yjshen324) [inline componentA/src/com/example/componentA/Util.java]: 22222

File: componentA/src/com/example/componentA/Util.java [hunk 5]
```diff
@@ -1,7 +1,7 @@
 package com.example.componentA;
```
Comment (yjshen324) [inline componentA/src/com/example/componentA/Util.java]: AAAAA

Comment (yjshen324) [inline componentA/src/com/example/componentA/Util.java]: BBBB

Comment (yjshen324) [inline componentA/src/com/example/componentA/Util.java]: AAA

File: componentA/src/com/example/componentA/Util.java (hunk 3 above)
Comment (yjshen324) [inline componentA/src/com/example/componentA/Util.java]: BBB

File: componentA/src/com/example/componentA/Util.java (hunk 5 above)
Comment (yjshen324) [inline componentA/src/com/example/componentA/Util.java]: AAA

File: componentA/src/com/example/componentA/Util.java [hunk 6]
```diff
@@ -1,7 +1,7 @@
...
 
 /* add some comments */
-public class Util {
+public class MyUtil {
 	public static boolean isBlank(String text) {
```
Comment (yjshen324) [inline componentA/src/com/example/componentA/Util.java]: @robinshine BBB

**Comment by yjshen324** [comment]
CCC

**Comment by yjshen324** [comment]
@robinshine DDD

**Pull request merged**